packages/integration-tests/src/define_backend_template_harness.ts

```diff
@@ -22,21 +22,33 @@ export const assertStableLogicalIds = (
 };
 
 /**
- * Wrapper around `defineBackend` that returns CDK Template objects for each backend resource
+ * Synthesizes deterministic `defineBackend` CDK templates for each backend resource
  *
  * Supplies stable CDK context values to the synth process for deterministic synth output
  */
-export const defineBackendTemplateHarness: DefineBackendTemplateHarness = <
+export const synthesizeBackendTemplates: SynthesizeBackendTemplates = <
```

Comment:
```suggestion
export const synthesizeBackendTemplates: SynthesizedBackendTemplates = <
```

if that's a return type.

Reply:
It's the type that the whole function implements, not the return type